app/instances.go

```diff
@@ -159,18 +159,18 @@ func (a *App) createTriggers(tConfigs []*trigger.Config, runner action.Runner) (
 							var ok bool
 							ref, ok = support.GetAliasRef("action", act.Ref)
 							if !ok {
-								return nil, fmt.Errorf("action '%s' not imported", act.Ref)
+								return nil, fmt.Errorf("action '%s' not imported in trigger [%s]'s handler [%s]", act.Ref, tConfig.Id, hConfig.Name)
 							}
 						}
 
 						actionFactory := action.GetFactory(ref)
 						if actionFactory == nil {
-							return nil, fmt.Errorf("action factory '%s' not registered", ref)
+							return nil, fmt.Errorf("action factory '%s' not registered in trigger [%s]'s handler [%s]", ref, tConfig.Id, hConfig.Name)
```

Comment:
fmt.Errorf("action factory '%s' referenced bytrigger [%s]'s handler [%s], has not been registered"